.github/workflows/appstore-build-publish.yml

```diff
@@ -162,7 +162,7 @@ jobs:
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: ${{ env.APP_NAME }}/build/artifacts/${{ env.APP_NAME }}.tar.gz
-          asset_name: ${{ env.APP_NAME }}-${{ env.APP_VERSION }}.tar.gz
+          asset_name: ${{ env.APP_NAME }}-${{ env.APP_VERSION }}.tar.gzK
```

Comment:
not that K :)

Reply:
just copied a file from https://github.com/nextcloud/.github/blob/master/workflow-templates/appstore-build-publish.yml